README.md

```diff
@@ -1,27 +1,47 @@
-# seneca-basic - a [Seneca](http://senecajs.org) plugin
+![Seneca](http://senecajs.org/files/assets/seneca-logo.png)
+> A [Seneca.js][] utility plugin
 
-### Seneca basic utility plugin. 
+# seneca-basic 
+[![Build Status][travis-badge]][travis-url]
+[![Gitter chat][gitter-badge]][gitter-url]
+
+[![js-standard-style][standard-badge]][standard-style]
 
 This plugin is included with the main seneca module and provides a
 small set of basic utility action patterns.
 
+- __Version:__ 0.3.0
+- __Tested on:__ [Seneca](//github.com/rjrodger/seneca) 0.7.2
+- __Node:__ 0.10, 0.12, 4
 
-### Support
+If you're using this module, and need help, you can:
 
-Current Version: 0.3.0
+- Post a [github issue](//github.com/rjrodger/seneca-basic/issues)
+- Tweet to [@senecajs](http://twitter.com/senecajs)
+- Ask on the [Gitter][gitter-url]
 
-Tested on: [Seneca](//github.com/rjrodger/seneca) 0.6.2
+seneca-basic's source can be read in an annotated fashion by,
+- running `npm run annotate`
+- viewing [online](http://rjrodger.github.io/seneca-basic/doc/basic.html)
 
-[![Build Status](https://travis-ci.org/rjrodger/seneca-basic.png?branch=master)](https://travis-ci.org/rjrodger/seneca-basic)
+The annotated source can be found locally at [./doc/seneca-basic.html]().
 
-[Annotated Source](http://rjrodger.github.io/seneca-basic/doc/basic.html)
+## Install
```

Comment:
Seneca basic is included by default with Seneca just like mem store. Take a look at the mem store readme for the install / explicit install stuff, add that here. https://github.com/senecajs/seneca-mem-store